packages/host/app/routes/render.ts

```diff
@@ -137,17 +180,52 @@ export default class RenderRoute extends Route<Model> {
     let lastModified = new Date(response.headers.get('last-modified')!);
     let doc: LooseSingleCardDocument | CardErrorsJSONAPI =
       await response.json();
+    let state = new TrackedMap<string, unknown>();
+    state.set('status', 'loading');
+
+    // the rendering of the templates is what pulls on the linked fields to load
+    // them. before the card templates are rendered there are no in-flight
+    // requests for linked fields. so in order to properly wait for the linked
+    // fields to load we must first render the /render/html route (preferably
+    // the isolated format), and then the store will start tracking the
+    // in-flight link requests. after the store settles the prerendered output
+    // will be ready to capture. this readyDeferred will let us know when the
+    // prerendered output is ready for capture.
+    let readyDeferred = new Deferred<void>();
+    let modelState: ModelState = {
+      state,
+      readyDeferred,
+      isReady: false,
+    };
```

Comment:
this is the meat of the new refactor and what is making our prerender stable now